### PR TITLE
newsweek-clone

### PR DESCRIPTION
**Pull request by billodiallo**
working on :
- header
- navbar
- footer
- mobile sm
- tablet md

MILESTONE 2
-feature stories
-sponsored insight
-top stories
-my turn
-culture and travel
- more stories
- the debate
- Opinion
free sign up

MILESTONE 3
-In the magazine
-Editor's Pick
-Sign Up Form
-Subscribe
-Sponsored Insights

### CODE REVIEW

**Review by SpaYco — APPROVED** [review]
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

- [ ] Make the nav matches the one in the original website ([Link to page](https://www.newsweek.com/)), make sure you match the spacing, borders, and layout 
- [ ] make the page centered and with space on the side, just like the original website ([Link to page](https://www.newsweek.com/))
- [ ] make the social icons on the footer on the left and `© 2021 NEWSWEEK DIGITAL LLC` next to the Logo, on mobile they will be each under the other. ([Link to page](https://www.newsweek.com/))
- [ ] Update the Live Demo link in your README.md
- [ ] Fix your linters.


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Invalid 🛑 

- [ ] in order to review the project, your linters needs to be fixed. (I know I reviewed the past milestone even though the linters are not fixed, but as of today repos needs to be fixed before a Code Reviewer can review a project/milestone)


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

File: styles/style.css [hunk 1]
```diff
@@ -1,40 +1,606 @@
...
+  .img-half {
+    height: 170px;
+  }
+
+  .f2 {
+    text-align: center;
+  }
+
+  .form-group {
+    display: flex;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `d-flex` Bootstrap class instead of this.

File: styles/style.css [hunk 2]
```diff
@@ -1,40 +1,606 @@
...
+  .info p {
+    margin-top: 15px;
+  }
+
+  .img-half {
+    height: 170px;
+  }
+
+  .f2 {
+    text-align: center;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `text-align-center` Bootstrap class instead of this.

File: styles/style.css [hunk 3]
```diff
@@ -1,40 +1,606 @@
...
+
+@media (max-width: 767.99px) {
+  .f1,
+  .flex-sm {
+    flex-direction: column;
+    text-align: center;
+  }
+
+  .info h3 {
+    position: absolute;
```
Comment (SpaYco) [inline styles/style.css:407]: - [ ] Use `position-absolute` Bootstrap class for this.

File: styles/style.css [hunk 4]
```diff
@@ -1,40 +1,606 @@
...
+
+  .f2 form input {
+    max-width: 100px;
+  }
+}
+
+@media (max-width: 767.99px) {
+  .f1,
+  .flex-sm {
+    flex-direction: column;
```
Comment (SpaYco) [inline styles/style.css:402]: - [ ] Use  `flex-direction-column` Bootstrap class for this.

File: styles/style.css [hunk 5]
```diff
@@ -1,40 +1,606 @@
...
+    text-align: center;
```
Comment (SpaYco) [inline styles/style.css:403]: - [ ] Use `text-align-center` Bootstrap class instead of this.

File: styles/style.css [hunk 6]
```diff
@@ -1,40 +1,606 @@
...
+
+.opinion .flex-sm {
+  border-bottom: 4px solid #d9d9d9;
+  margin-bottom: 10px;
+}
+
+.f2 {
+  border: 1px solid #ebebeb;
+  padding: 30px;
+  padding-right: 0;
```
Comment (SpaYco) [inline styles/style.css:372]: - [ ] Use `pr-0` Bootstrap class for this.

File: styles/style.css [hunk 7]
```diff
@@ -1,40 +1,606 @@
...
+
+.block-title {
+  border-top: 1px solid black;
+  color: red;
+  font-size: 1.4rem;
+  padding-bottom: 11px;
+}
+
+.flex-sm {
+  display: flex;
```
Comment (SpaYco) [inline styles/style.css:361]: - [ ] Use `d-flex` Bootstrap class for this.

File: styles/style.css [hunk 8]
```diff
@@ -1,40 +1,606 @@
...
+  padding-left: 30px;
+}
+
+.info h3,
+p {
+  line-height: 28px;
+}
+
+.img-half {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css:347]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 9]
```diff
@@ -1,40 +1,606 @@
...
+  align-items: center;
 }
 
-.copyright {
-  padding-top: 27px;
-  margin-left: 14px;
-}
\ No newline at end of file
+.upper-image {
+  position: absolute;
```
Comment (SpaYco) [inline styles/style.css:332]: - [ ] Use `position-absolute` Bootstrap class for this.

File: styles/style.css [hunk 10]
```diff
@@ -1,40 +1,606 @@
...
+
+.image-subscribe {
+  position: relative;
+  padding-right: 90px;
+  padding-bottom: 15px;
+}
+
+.col-1 {
+  padding: 30px;
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css:325]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 11]
```diff
@@ -1,40 +1,606 @@
...
   background-color: #f4f5f6;
-  color: #333;
+  min-width: 100%;
+  align-items: center;
```
Comment (SpaYco) [inline styles/style.css:328]: - [ ] Use `align-items-center` Bootstrap class for this.

File: styles/style.css [hunk 12]
```diff
@@ -1,40 +1,606 @@
...
+  margin-right: auto;
+  margin-left: auto;
+} */
+
+.f1 {
+  display: flex;
+}
+
+.image-subscribe {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css:318]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 13]
```diff
@@ -1,40 +1,606 @@
...
+
+.col-sm-4 {
+  padding: 0 15px;
+}
+
+/* .container-fluid {
+  max-width: 1310px;
+  margin-right: auto;
+  margin-left: auto;
+} */
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Remove commented code

File: styles/style.css [hunk 14]
```diff
@@ -1,40 +1,606 @@
...
+  padding-bottom: 10px;
+}
+
+.feature ul {
+  padding-left: 0;
+}
+
+.container {
+  margin: 0;
+  padding: 0;
```
Comment (SpaYco) [inline styles/style.css:310]: - [ ] use `m-0` and `p-0` Bootstrap classes for these.

File: styles/style.css [hunk 15]
```diff
@@ -1,40 +1,606 @@
...
+  margin-bottom: 15px;
+}
+
+.flex-xs {
+  display: flex;
+  padding-bottom: 10px;
+}
+
+.feature ul {
+  padding-left: 0;
```
Comment (SpaYco) [inline styles/style.css:305]: - [ ] Use `pl-0` Bootstrap class for this.

File: styles/style.css [hunk 16]
```diff
@@ -1,40 +1,606 @@
...
+  height: 110px;
+}
+
+.border-bottom {
+  border-bottom: 1px solid black;
+  margin-bottom: 15px;
+}
+
+.flex-xs {
+  display: flex;
```
Comment (SpaYco) [inline styles/style.css:300]: - [ ] Use `d-flex` Bootstrap class for this.

File: styles/style.css [hunk 17]
```diff
@@ -1,40 +1,606 @@
...
+    url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain
+  no-repeat #f72210;
+}
+
+.wrap {
+  position: relative;
+}
+
+.category {
+  position: absolute;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `position-absolute` Bootstrap class for this.

File: styles/style.css [hunk 18]
```diff
@@ -1,40 +1,606 @@
...
+  width: 29px;
+  height: 28px;
+  margin-right: 10px;
+  background:
+    url(https://g.newsweek.com/www/images/logo-n1.svg) center/contain
+  no-repeat #f72210;
+}
+
+.wrap {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 19]
```diff
@@ -1,40 +1,606 @@
...
+  margin-bottom: 50px;
+}
+
+.editor-section .row article {
+  margin-bottom: 20px;
+}
+
+.header-editor {
+  border-bottom: 1px solid gray;
+  display: flex;
```
Comment (SpaYco) [inline styles/style.css:263]: - [ ] Use `d-flex` Bootstrap class for this.

File: styles/style.css [hunk 20]
```diff
@@ -1,40 +1,606 @@
...
+  justify-content: center;
```
Comment (SpaYco) [inline styles/style.css:264]: - [ ] Use `justify-content-center` Bootstrap class for this.

File: styles/style.css [hunk 21]
```diff
@@ -1,40 +1,606 @@
...
+  margin-top: 10px;
+}
+
+@media (max-width: 767.99px) {
+  .upper-part {
+    flex-direction: column;
+  }
+
+  .flex-sm-3 {
+    display: flex;
```
Comment (SpaYco) [inline styles/style.css:241]: - [ ] Use `d-flex` Bootstrap class for this.

File: styles/style.css [hunk 22]
```diff
@@ -1,40 +1,606 @@
...
+    flex-direction: column;
```
Comment (SpaYco) [inline styles/style.css:242]: - [ ] Use `flex-direction-column` Bootstrap class for this.

File: styles/style.css [hunk 23]
```diff
@@ -1,40 +1,606 @@
...
+    width: 100%;
+    padding: 0 15px 0 15px;
+    align-items: center;
```
Comment (SpaYco) [inline styles/style.css:245]: - [ ] Use `align-items-center` Bootstrap class for this.

File: styles/style.css [hunk 24]
```diff
@@ -1,40 +1,606 @@
...
+    justify-content: center;
```
Comment (SpaYco) [inline styles/style.css:246]: - [ ] - [ ] Use `justify-content-center` Bootstrap class for this.

File: styles/style.css [hunk 25]
```diff
@@ -1,40 +1,606 @@
...
+}
+
+.bottom-part h5,
+.top-part h5 {
+  margin-top: 10px;
+}
+
+@media (max-width: 767.99px) {
+  .upper-part {
+    flex-direction: column;
```
Comment (SpaYco) [inline styles/style.css:237]: - [ ] - [ ] Use `flex-direction-column` Bootstrap class for this.

File: styles/style.css [hunk 26]
```diff
@@ -1,40 +1,606 @@
...
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
+  font: bold 14px/32px "Roboto Condensed", sans-serif;
+  text-align: center;
+  margin: 6px auto;
+}
+
+.ai-c {
+  align-items: center;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] - [ ] Use `align-items-center` Bootstrap class for this.

File: styles/style.css [hunk 27]
```diff
@@ -1,40 +1,606 @@
...
+  background-color: #d9d9d9;
+  height: 2px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: calc(50% - 2px);
+}
+
+.c-vs {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 28]
```diff
@@ -1,40 +1,606 @@
...
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
```
Comment (SpaYco) [inline styles/style.css:171]: - [ ] Use `rounded-circle` Bootstrap class for this.

File: styles/style.css [hunk 29]
```diff
@@ -1,40 +1,606 @@
...
+.debate-border {
+  border-top: 4px solid #d9d9d9;
+}
+
+.debate-border-bottom {
+  border-bottom: 4px solid #d9d9d9;
+}
+
+.deco {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 30]
```diff
@@ -1,40 +1,606 @@
...
+.nav-bottom span {
+  padding: 10px 20px;
+  font-weight: bold;
+}
+
+.nav-bottom a {
+  text-decoration: none;
+  color: #aaa;
+  padding: 10px 20px;
+  font-weight: bold;
```
Comment (SpaYco) [inline styles/style.css:39]: - [ ] Use `font-weight-bold` Bootstrap class for this.

File: styles/style.css [hunk 31]
```diff
@@ -1,40 +1,606 @@
...
+  margin-left: -15px;
+  width: 1270px;
+  background-color: #f4f5f6;
+  border-top: 1px solid lightgray;
+  border-bottom: 1px solid lightgray;
+}
+
+.nav-bottom span {
+  padding: 10px 20px;
+  font-weight: bold;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `font-weight-bold` Bootstrap class for this.

File: styles/style.css [hunk 32]
```diff
@@ -1,40 +1,606 @@
...
+  max-width: 1310px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.nav-sticky {
+  background-color: red;
+  width: 100%;
+  max-width: 1270px;
+  position: fixed;
```
Comment (SpaYco) [inline styles/style.css:9]: - [ ] Use `position-fixed` Bootstrap class for this.

File: styles/style.css [hunk 33]
```diff
@@ -1,40 +1,606 @@
...
+  top: 0;
+  z-index: 4;
+}
+
+.not-sticky {
+  width: 1200px;
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css:16]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 34]
```diff
@@ -1,40 +1,606 @@
...
+  z-index: 9;
+}
+
+.nav-color {
+  position: relative;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `position-relative` Bootstrap class for this.

File: styles/style.css [hunk 35]
```diff
@@ -1,40 +1,606 @@
...
+  margin-top: 3px;
+  width: 100%;
+  height: auto;
+  font-weight: bolder;
+}
+
+.navbar > div > button {
+  width: 110px;
+  height: auto;
+  font-weight: bolder;
```
Comment (SpaYco) [inline styles/style.css]: - [ ] Use `font-weight-bolder` Bootstrap class for this.

**Review by SpaYco — APPROVED** [review]
# Milestone Approved ✔️ 

Milestone approved, however, there are things that you need to change, you can either make the changes now, or you keep them to the last milestone.

**General**
- [ ] Fix the colors in the website and make it match the original website

**Tablet**
- [ ] The right section (Debate, Opinion, etc...) just disappears on the right side of the page, and I can see it when I scroll there. it should instead be at the bottom after the other 2 sections

**Mobile**

- [ ] Make sure you follow the right order for mobile

<table>
    <tbody>
        <tr>
            <td>TOP STORY</th>
        </tr>
        <tr>
            <td>FEATURED STORIES</th>
        </tr>
        <tr>
            <td>THE DEBATE</th>
        </tr>
        <tr>
            <td>OPINION</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
        <tr>
            <td>MORE STORIES</th>
        </tr>
        <tr>
            <td>LATEST NEWS</th>
        </tr>
        <tr>
            <td>MY TURN</th>
        </tr>
        <tr>
            <td>CULTURE & TRAVEL</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
        <tr>
            <td>FREE SIGN UP</th>
        </tr>
        <tr>
            <td>SPONSORED INSIGHT</th>
        </tr>
    </tbody>
</table>


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] the Subscribe / Newsletter sections aren't there, please fix them
- [ ] fix the problems mentioned in the previous review(s)
- [ ] check my comments above in the previous reviews to fix the CSS properties that can be turned into bootstrap classes, make sure to double-check in case I miss some.

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by billodiallo** [comment]
-TABLET and Large screen fixed.

ordered in Mobile 

TOP STORY
--
FEATURED STORIES
THE DEBATE
OPINION
SPONSORED INSIGHT
MORE STORIES
LATEST NEWS
MY TURN
CULTURE & TRAVEL
SPONSORED INSIGHT
FREE SIGN UP
SPONSORED INSIGHT

--MATCHING COLORS like original.
-- changed some CSS code using  bootstrap

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] The navbar is smaller than the rest of the page, make them equal
- [ ] you still have a lot of CSS properties that can be turned into bootstrap classes, please fix **all** the ones I mentioned above, I can see that only 3 of what I mentioned got fixed.

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Comment by billodiallo** [comment]
Dear @Spayco 
We fixed the navbar issue and change CSS code that are changeable in to bootstrap, the rest if it affect all the page and give more time for debugging. Thanks for your review.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @ . Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 
- [ ] Branch name should be descriptive and meaningful, please edit the branch name and update the PR with the new branch name, DO NOT close the PR and open a new one USE THE SAME PR just update the branch name. Example of good branch name: `newsweek-clone`
- [ ] The issues still persisnt on the navbar
image
- [x] The logo on the footer doesn't match original
image
- [ ] Editors Pick section should change to a two column layout on tablet view
image
- [ ] Navbar z-index should be fix so this doesn't happen
image
image
- [ ] Alignment of the text
image
- [ ] Use bootstrap classes instead of custom CSS rules. Review previous inline observations in `style.css` file and remove rules that have a bootstrap class counterpart.
  - [ ] text decoration
  - [ ] text color
  - [ ] display
  - [ ] position
  - [ ] text transform
  - [ ] font weight bold
  - [ ] text decoration
  - [ ] text align
  - [ ] flex direction
  - [ ] align items
  - [ ] justify content
  - [ ] padign right 0

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

<div>
  <div style="width:50px; height: 50px" align="center">
    <a href="https://avatars.githubusercontent.com/u/25774518?s=400&u=ae20ddf2d6fb3cae9e2f4044afe3b93d19e5b69e&v=4" alt="Julio Añoveros" title="@jubaan" width="100px" heigh="100px">
    </a>
  </div>
  </br>
  <p align="center">
    Julio Añoveros | @jubaan</br></br>
    <b>Full-stack Web Developer</b> and</br>
    Code Reviewer at <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a>:rocket:
  </p>
  <p align="center">
    <b>Feel stuck?</b></br>
    First breath... just do it.</br>
    Take 15-20 min away from the computer,</br>
    stretch, meditate, do some exercise, drink water,</br>
    or do whatever is necessary to distract your mind</br> 
    from the obstacle.</br></br> 
    Sometimes you just need to get away</br>
    to see things with a new perspetive.
  </p>
  <p align="center">
    <b>Need help?</b></br>
    Send me a DM in <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> Students Slack,</br>
    you'll find me as @jubaan.
  </p>
  </br>
  <p align="center">
    <q>A <b>great programer</b> is not the one that knows it all,</br>
    but the one that ask a lot of question.</q></br></br>
    <b>Don't hesitate to ask for help!</b>
  </p>
</div>

**Pull request closed without merging**